server/routes/soonBusStop.js

```diff
@@ -0,0 +1,127 @@
+import { calcCWorCCW } from "./direction.js"
+const busStops = require('./bus-stops.json')
+const defaultDatabase = require('./firebase.js')
+
+let bussesRef;
```

Comment:
Keep global variables inside exported functions. Weird stuff might happen if two of the exported functions are called at the same time